### ISSUE TITLE
[1.16] New recipes doesn

### ISSUE THREAD

**Issue by hodo1962** [CraftTweaker/CraftTweaker#1045]
#### Intro:
Please note, you will be treated exactly like any other person who submits a bug report, saying this, if you decide to delete this template entirely, your issue will be closed. If you do not provide relevant information when reporting an issue (for example having a script issue and not providing the scripts or the crafttweaker.log file) this issue will be closed. you may now delete this section and continue on with the report.

#### Issue Description:
Note: If this bug occurs in a modpack, please report this to the modpack author. Otherwise, delete this line and add your description here. If you are using an unnofficial version of Minetweaker or Crafttweaker please report it to the person who gave you the version. Also please look at the other issues to make sure that you are not creating a duplicate.

#### crafttweaker.log file (Please pastebin.com or gist.github.com your file, posting an unpasted or ungist'd file will automatically close this issue. We are **NOT** going to download files, do **NOT** upload your file to this issue, please give us a link to a paste / gist of the file.):

____
#### Affected Versions (Do *not* use "latest"):

- Minecraft:
- Forge:
- Crafttweaker: 
- Using a server: 
- If yes, does the client have the exact same scripts?

#### Your most recent log file where the issue was present (Please pastebin.com or gist.github.com your file, posting an unpasted or ungist'd file will automatically close this issue. We are **NOT** going to download files, do **NOT** upload your file to this issue, please give us a link to a paste / gist of the file.):

[pastebin/gist/etc link here]

**Comment by hodo1962** [comment]
I've tried to make a z-script with a new shapeless recipe. The recipe should craft the Ingots from one mod into material from another mod. Here is the line from the script:

_craftingTable.addShapeless("uranite_conversion1", <item:powah:uraninite_raw>, [<item:mekanism:ingot_uranium>, <item:mekanism:ingot_uranium>]);_

 After restart I could see the new recipe in JEI, but it didn't work in the crafting table.
I've found out a workaround:
1. Deleting all recipes by typing: /recipe take *
2. Giving all recipes by typing: /recipe give * 

After that the new recipe worked.

**Affected Versions:**
Minecraft: 1.16.1
Forge: 32.0.107
Crafttweaker: 1.16.1-7.0.0.23
Using a server: no

(i wasn't fast enough typing)
